core/src/test/java/org/springframework/security/authorization/method/PostAuthorizeAuthorizationManagerTests.java

```diff
@@ -51,7 +51,7 @@ public void setExpressionHandlerWhenNotNullThenSetsExpressionHandler() {
 		MethodSecurityExpressionHandler expressionHandler = new DefaultMethodSecurityExpressionHandler();
 		PostAuthorizeAuthorizationManager manager = new PostAuthorizeAuthorizationManager();
 		manager.setExpressionHandler(expressionHandler);
-		assertThat(manager).extracting("expressionHandler").isEqualTo(expressionHandler);
+		assertThat(manager).extracting("registry").extracting("expressionHandler").isEqualTo(expressionHandler);
```

Comment:
The testing on the servlet side is quite a bit more extensive. It seems we'd want to add a number of tests to `ReactiveMethodSecurityConfigurationTests` that are similar in depth to those in `PrePostMethodSecurityConfigurationTests`.

Reply:
Done.